### PR TITLE
Vcon 221 splash page

### PR DESCRIPTION
**Pull request by DaceyTom2**
### Description
Added splash page to root path

### Additional Notes
app layout was moved into pages to allow for splash screen
design is a mix between canada.ca and alpha with a heavier focus on canada.ca
navigation to home must be done using language /en or /fr
background image was taken from Alpha but can be changed

### CODE REVIEW

**Review by TishamIslam — CHANGES_REQUESTED** [review]
Only the bit about tailwind _needs_ to be changed. Looks really amazing so far, I can see where it's similar to canada.ca and Alpha site, but also different. I don't think it's absolutely necessary to change any/all of the following points, but I think they're important to consider and maybe discuss with Thomas or some other people:

- Alpha site has that url thing on the splash page, while canada.ca doesn't. That makes sense since Alpha isn't the main gov site. However, ours doesn't have that url identifier and I'm not sure if Alpha is supposed to be the trend here or if it's supposed to be the exception, and if the other sites (Life Journeys etc.) will follow along. Something to discuss with the other teams I think.

- The button widths for English and French are different widths on this splash page, whereas Alpha and canada.ca have the same sizes. It'd make sense to change it to fit those conventions, but isn't that big of a deal. 

- I noticed that Canada.ca has shadows on their buttons, would be cool if we could also have shadows.

**Review by shewood — COMMENTED** [review]
looks amazing.
Would there be a higher res image. this one looks quite blurry. 
Not sure about the addition of the colours. Thomas has his own choices for the VC site.

File: tailwind.config.js [hunk 1]
```diff
@@ -59,6 +63,10 @@ module.exports = {
         "vh-1/4": "calc(100vh / 4)",
         "vh-1/5": "calc(100vh / 5)",
       },
+      width: {
+        "300px": "300px",
+        "500px": "500px",
```
[the remark this answers is not in the document]
Comment (shewood) [reply tailwind.config.js:68]: agreed

**Comment by neilmispelaar** [comment]
This looks so amazing. 

I'm just doing a quick comparison of GCWEB and the current implementation and will get back to you with the delta.

**Comment by neilmispelaar** [comment]
#### GCWEB
https://wet-boew.github.io/GCWeb/
Screen Shot 2021-07-14 at 11 23 06 AM

#### Current implementation
Screen Shot 2021-07-14 at 11 23 14 AM

Background image aside (that doesn't matter) - it looks like GCWeb added:
-  a weird bottom border colour on the two buttons
- is slightly higher up on the page

Not sure if these are honestly even worth implementing - but thought I would point them out. 

Just reviewing the routes and rest of the PR now...

**Comment by neilmispelaar** [comment]
Oh shoot - so sorry @TishamIslam - I'm realizing that you made literally the exact same points above. This is what happens when you don't read good.

File: tailwind.config.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (DaceyTom2) [reply tailwind.config.js]: Fixed!

File: src/views/NotFound.vue [hunk 2]
```diff
@@ -1,11 +1,36 @@
 <template>
-  <section>
-    <div class="text-4xl">Not found</div>
-  </section>
+  <!-- skip nav component -->
```
[the remark this answers is not in the document]
Comment (DaceyTom2) [reply src/views/NotFound.vue:2]: I had to move the layout from app.vue but this is intentional to keep the header and footer on the 404 page

**Comment by DaceyTom2** [comment]
@TishamIslam 
I did mention to Thomas Hoy I was leaving out the alpha link and he was okay with it. Personally I am not a fan of it since it is a splash page to our page and can be skipped if language is already provided. I do agree we should probably keep as close as possible with other teams.

The buttons are updated to match sizes with Alpha and Canada.ca

Shadows have been added but tailwind shadows aren't as defined

@shewood 
The image is actually quite high res and alpha/Canada.ca/this PR adds a blur effect. I don't understand why but I followed suit...

@neilmispelaar 
Alpha has the box in the middle and canada.ca has it in a static position just below the top. Personally I like it in the middle but can change if desired

**Comment by neilmispelaar** [comment]
#### Domain in box 
- I will echo that I think adding the domain to the box is silly. 

#### Position of box 
- I'm not going to lose sleep over it either way.

**Comment by neilmispelaar** [comment]
Slide bug on the alpha site in mobile landscape if you are copy/pasting code

IMG_4421
IMG_4420

**Comment by neilmispelaar** [comment]
Sorry - it's doing it on ours as well:

IMG_4422

**Comment by neilmispelaar** [comment]
If you do have a fix - the Alpha site could probably also benefit from it: 

https://github.com/DTS-STN/Alpha-Site/issues/191

**Review by TishamIslam — APPROVED** [review]
The new additions are looking great, you could change the names of the added tailwind sizes to fit the convention (would be 34.5, 75 and 125 in order of least to greatest) but aside from that , I approve.

**Comment by DaceyTom2** [comment]
@TishamIslam I was following along with Alpha naming. I'll leave it for now.

@neilmispelaar Good catch! That was tough to fix but is much nicer now

**Comment by neilmispelaar** [comment]
Ultra small WCAG issue:

### Description

French words on the page should be identified using the `lang` attribute set to `fr`.

- There are two French words on the page
  - The French button
  - The Aviz link 
- https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts.html

How to fix: 

```
<router-link
  ...
  to="/fr"
  >
  <span lang="fr">
     Français
  </span>
</router-link>
```

**Pull request merged**